xml/Microsoft.Extensions.Logging.Internal/FormattedLogValues.xml

```diff
@@ -38,7 +38,7 @@
     </Interface>
   </Interfaces>
   <Docs>
-    <summary>A collection of log values formatted with options supported by <code>String.Format</code>. This type also enables using {NamedformatItem} in the format string.</summary>
+    <summary>A collection of log values formatted with options supported by <c>String.Format</c>. This type also enables using {NamedformatItem} in the format string.</summary>
```

Comment:
Should this one (and the other inline ones) be a cref to https://docs.microsoft.com/en-us/dotnet/api/system.string.format?view=net-5.0?